src/2980R0_a_motivation_scope_and_plan_for_a_physical_quantities_and_units_library.md

```diff
@@ -235,6 +235,21 @@ As a result, companies either use really simple and unsafe numeric wrappers or a
 and just use `double` to express quantity values which lead to safety issues by accidentally using
 values representing the wrong quantity or having an incorrect unit.
 
+## Extensibility
+
+Many applications of a quantity and units library may need to operate on
+a combination of standard (e.g. time in seconds or distance in metres)
+and domain-specific quantities and units.  The complexity of developing
```

Comment:
For clarity, before giving examples, it should be clear that "standard" refers to "standard quantities and units".
So I suggest this rewrite:
```suggestion
a combination of quantities and units
that are standard (e.g. time in seconds or distance in metres)
and domain-specific.
The complexity of developing
```